src/cpp/session/modules/SessionDataImportV2.R

```diff
@@ -13,6 +13,27 @@
 #
 #
 
+.rs.addFunction("path_relative_to_working_dir", function(target)
+{
+   if (is.null(target))
+   {
+      target
+   }
+   else
+   {
+      currentPath <- paste(getwd(), .Platform$file.sep, sep = "")
```

Comment:
I think on Windows we still want to use `/` as the path separator -- `\` is mainly reserved for paths that need to be sent to system utilities / the shell. (In addition, I think `getwd()` will typically return a path using `/` as the separator)